crypto/keyring/keyring_linux.go

```diff
@@ -0,0 +1,43 @@
+//go:build linux
+// +build linux
+
+package keyring
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/99designs/keyring"
+	"github.com/cosmos/cosmos-sdk/codec"
+)
+
+// Linux-only backend options.
+const BackendKeyctl = "keyctl"
+
+func newKeyctlBackendConfig(appName, _ string, _ io.Reader) keyring.Config {
+	return keyring.Config{
+		AllowedBackends: []keyring.BackendType{keyring.KeyCtlBackend},
+		ServiceName:     appName,
+		KeyCtlScope:     "user",
+		KeyCtlPerm:      0x3f3f0000,
```

Comment:
Can you add a clarification on which are the granted permissions here?

Reply:
I am doing more work to provide users with sensible perms/scope options

Reply:
By default, `user` and `session` keyring give access to the owner (i.e. the creator) only